addons/l10n_ar/models/account_move.py

```diff
@@ -405,6 +410,24 @@ def _l10n_ar_get_invoice_totals_for_report(self):
 
             tax_totals["detail_ar_tax"] = list(detail_info.values())
 
+        if self._is_refund_invoice():
+            if 'amount_total' in tax_totals:
+                tax_totals['amount_total'] = -tax_totals['amount_total']
+                tax_totals['formatted_amount_total'] = formatLang(self.env, tax_totals['amount_total'], currency_obj=self.currency_id)
+            if 'amount_untaxed' in tax_totals:
+                tax_totals['amount_untaxed'] = -tax_totals['amount_untaxed']
+                tax_totals['formatted_amount_untaxed'] = formatLang(self.env, tax_totals['amount_untaxed'], currency_obj=self.currency_id)
+            for group in tax_totals.get('groups_by_subtotal', {}).values():
+                for subtotal in group:
+                    subtotal['tax_group_amount'] = -subtotal['tax_group_amount']
+                    subtotal['tax_group_base_amount'] = -subtotal['tax_group_base_amount']
+                    subtotal['formatted_tax_group_amount'] = formatLang(self.env, subtotal['tax_group_amount'], currency_obj=self.currency_id)
+                    subtotal['formatted_tax_group_base_amount'] = formatLang(self.env, subtotal['tax_group_base_amount'], currency_obj=self.currency_id)
+            for subtotal in tax_totals.get('subtotals', []):
+                subtotal['amount'] = -subtotal['amount']
+                if 'formatted_amount' in subtotal:
+                    subtotal['formatted_amount'] = formatLang(self.env, subtotal['amount'], currency_obj=self.currency_id)
```

Comment:
@jue-adhoc 
maybe can be improve:
* only change it the  keys that start with formatted_
* with a generic approach  iterating inside the dictionary to change all the keys that start with formatted_. Will be less lines but difficult to track if we have rrror

I will not do any change until get tecnical feedback from odoo tecnical team suggestions